package.json

```diff
@@ -1201,6 +1218,11 @@
         "view": "confluent-flink-database",
         "contents": "No UDFs found.\n[Generate a new Flink UDF from template](command:confluent.artifacts.scaffold)",
         "when": "config.confluent.flink.artifacts && confluent.ccloudConnectionAvailable && confluent.flinkDatabaseSelected && confluent.flinkDatabaseViewMode == 'UDFs'"
+      },
+      {
+        "view": "confluent-flink-database",
+        "contents": "No Flink database contents matches search string. [Clear search](command:confluent.flink.database.search.clear) or [try a new search](command:confluent.flink.database.search).",
```

Comment:
Should we split this based on which view mode is active? 
- `No Flink artifacts match search string` for `confluent.flinkDatabaseViewMode == 'UDFs'`
- `No Flink UDFs match search string` for `confluent.flinkDatabaseViewMode == 'artifacts'`?